core/arch/arm/pta/se_api_self_tests.c

```diff
@@ -77,30 +77,6 @@
  * Trusted Application Entry Points
  */
 
-static TEE_Result create_ta(void)
```

Comment:
Ah, ok, i understand. Ok, I keep the create_ta() handler for the self test pTA with its trace.